packages/blunderbuss/README.md

```diff
@@ -12,18 +12,28 @@ Blunderbuss randomly assigns from a set of users specified in the config file lo
 assign_issues:
   - issue_assignee1
   - issue_assignee2
+  - by:
```

Comment:
Thinking about this, two potential alternatives come to mind: 

First - we could just flatten it to "by_labels", and have it be a list of "config objects". This still feels (maybe?) a bit misleading since the "by_labels" have precedence even when mixed in, and is a bit at odds because the "by_labels" configs have multiple assignees, but the "default" assignees are all listed separately. Example:
```yaml
assign_issues:
  - issue_assignee1
  - issue_assignee2
  - by: 
     - labels: ['label'] 
        to: ['assignee']
```

Second - we could pull it out into a new top level config - "assign_issues_on_labels" that has the same signature has "by". This might make it easier to think about "by_labels" having precedence over the "default" assignees. Example: 
```yaml
assign_issues_on_labels:
  - labels: ['label'] 
     to: ['assignee']
```

Third - we use a map with separate components for "by" and "default", which might be clearer:
```yaml
assign_issues:
    by: 
     - labels: ['label'] 
        to: ['assignee']
    default:
       - assignee
       - assignee
```
The downside to this is it's a breaking change, but we could introduce versioning to avoid breaking the current configs (but that's probably a bit of work).

what do you think?

Reply:
I like the second option (top level config) the most from those. But, I'm not sure it's better than the current setup. I started with `by_labels`, but wanted to keep it more general in case there were other options in the future. I'm not too concerned about the precedence being clear, but I could be wrong. Definitely not worth a breaking change to me.

Reply:
What did you have in mind for other options, and how did you intend to handle precedence for those?

Would it be better to just have the top level section of "issues_by:" and "prs_br:"?

Reply:
The config for where to look for labels might be in a different place (Sloth). That's the main expansion I have in mind.

For precedence, specific > general. If a `by` config matches, the candidates are added to the random pool. If no config matches, the general candidates are used, if any. That's not critical for this PR, though.

I'd be fine with a top-level `issues_by` or keeping it as-is. I don't see a large benefit of one or the other.

Reply:
Done (as `assign_issues_by`).